boards/arm/qemu_cortex_a53/qemu_cortex_a53.yaml

```diff
@@ -10,4 +10,4 @@ ram: 128
 testing:
   default: true
   ignore_tags:
-    - interrupt
+    - nmi
```

Comment:
Adding `nmi` to the ignored tags list should not be necessary.

The Shippable failure will go away once you cherry-pick this commit into this PR (to add the missing filter in `tests/arch/arm/arm_runtime_nmi`): https://github.com/zephyrproject-rtos/zephyr/pull/23636/commits/669069118538a091c60267000bcdbafd5fbf13e6

Reply:
ouch, missed that.